addons/csw/functions/fnc_proxyWeapon.sqf

```diff
@@ -21,6 +21,11 @@
 params ["_vehicle", "_turret", "_needed", "_emptyWeapon"];
 TRACE_4("proxyWeapon",_vehicle,_turret,_needed,_emptyWeapon);
 
+// addWeaponTurret/removeWeaponTurret need to be executed where turret is local
+if !(_vehicle turretLocal _turret) exitWith {
+    TRACE_1("turret not local",_vehicle);
```

Comment:
```suggestion
    WARNING_1("turret not local %1",_this);
```

Reply:
Edited your suggestion to be a bit more verbose.